diff --git a/node_modules/formik/dist/types.d.ts b/node_modules/formik/dist/types.d.ts
index aeb2b19..82f0c2d 100644
--- a/node_modules/formik/dist/types.d.ts
+++ b/node_modules/formik/dist/types.d.ts
@@ -191,7 +191,7 @@ export interface SharedRenderProps<T> {
     /**
      * Field component to render. Can either be a string like 'select' or a component.
      */
-    component?: string | React.ComponentType<T | void>;
+    component?: string | React.ComponentType<T | any>;
     /**
      * Render prop (works like React router's <Route render={props =>} />)
      */
